### ISSUE TITLE
ntsync support

### ISSUE THREAD

**Issue by dougg0k** [CachyOS/proton-cachyos#3]
Hi,

Any chance of adding latest ntsync optionally (behind a flag) on proton-cachyos?

[ntsync5-mainline.patch](https://github.com/Frogging-Family/wine-tkg-git/blob/master/wine-tkg-git/wine-tkg-patches/misc/fastsync/ntsync5-mainline.patch)

https://github.com/ValveSoftware/Proton/issues/7501#issuecomment-2495681848

**Comment by dougg0k** [comment]
@ptr1337

**Comment by loathingKernel** [comment]
Being worked on here https://github.com/CachyOS/wine-cachyos/tree/cachyos_staging/ntsync7

**Comment by zqpvr** [comment]
would this work for proton?

**Comment by ptr1337** [comment]
> would this work for proton?

yes

You can find a precompiled proton here:
https://share.cachyos.org/proton/proton-cachyos-ntsync-1%3A9.0.20241206-4-x86_64_v3.pkg.tar.zst

**Comment by dougg0k** [comment]
Any idea when that is going to be stabilized and released?

What flag(s) there will be to activate it?

**Comment by loathingKernel** [comment]
No plan yet, and it is not that interesting anyways given testing showed that results are a coin toss in terms of this port, it suffers from issues that are not present in the original patchset.

**Comment by dougg0k** [comment]
Fixable issues? What issues?

**Comment by dougg0k** [comment]
Just saw the release https://github.com/CachyOS/proton-cachyos/releases/tag/cachyos-9.0-20250102-slr but there is no update in the mirrors.

**Comment by ptr1337** [comment]
> Just saw the release [`cachyos-9.0-20250102-slr` (release)](https://github.com/CachyOS/proton-cachyos/releases/tag/cachyos-9.0-20250102-slr) but there is no update in the mirrors.

This is a testing release for steam runtime only.

**Comment by dougg0k** [comment]
I see, I asked because I saw a branch with ntsync7 in the same date.

**Comment by dougg0k** [comment]
@ptr1337 is there a public testing mirror for the ntsync7 branch version?

**Comment by loathingKernel** [comment]
No, there is not, and there won't be. We do test sometimes with people on our discord server or per-case. That being said, the code is in the repo, feel free to build it. It has numerous issues right now and I do not want to put it into a release.

**Comment by dougg0k** [comment]
I see. Can you list the issues, so others can be aware?

**Comment by dougg0k** [comment]
https://discuss.cachyos.org/t/ntsync-in-latest-proton-cachyos-wine-cachyos/5254/4
